projects/observability/src/shared/components/card-list/card-list.component.test.ts

```diff
@@ -62,4 +62,43 @@ describe('Card List component', () => {
     expect(spectator.query('.selected-card')).not.toBe(cardElements[1]);
     expect(spectator.query('.selected-card')).not.toBe(cardElements[2]);
   });
+
+  test('should apply grouped style class', () => {
+    const data = [
+      {
+        name: 'first'
+      },
+      {
+        name: 'second'
+      },
+      {
+        name: 'third'
+      }
+    ];
+
+    spectator = createHost(
+      `
+    <ht-card-list [mode]="mode">
+      <ht-card-container *ngFor="let cardData of this.data; first" showGroupedStyle="{{cardData === first ? true: false}}">
```

Comment:
confusing way to write a test - can we just stick this boolean on the data object? (or at least remove the `? true : false` - my arch nemesis)

Reply:
I only want to set it for first card so that I can test that the class is only applied to one card.

Reply:
Right - I meant something like below. Make the test explicit rather than logic-based.
```ts
      {
        name: 'first',
        grouped: true
      },
      {
        name: 'second'
      },
      {
        name: 'third'
      }
```

Reply:
okay